### PR TITLE
SEO-2024 Blazor Progress Button External Linking

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @ChristoIssac ,

Please review the changes made for Blazor Progress Button External Linking.

Regards,
Yvone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,64 @@
...
+* Info
+* Warning
+* Danger
+
+## Custom Progress Button
+
+Blazor Progress buttons can be customized in multiple ways. You can choose from different button sizes, default and small, as required in your application.
+
+## Related links
+[Learn More about Blazor ListView](https://www.syncfusion.com/blazor-components/blazor-progress-button?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples)
```
Comment (Girija-Rajendran) [inline README.md]: is this ListView control?
image

File: README.md [hunk 2]
```diff
@@ -1,15 +1,64 @@
...
...w.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples)), and desktop development ([WinForms](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples), [WPF](https://www.syncfusion.com/wpf-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples), [WinUI](https://www.syncfusion.com/winui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples), and [MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```
Comment (Girija-Rajendran) [inline README.md]: [.NET MAUI(Preview)](....)

File: README.md (hunk 1 above)
Comment (Yvone-Atieno) [reply README.md]: resolved

File: README.md (hunk 2 above)
Comment (Yvone-Atieno) [reply README.md]: resolved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**